Externals.cfg

```diff
@@ -12,7 +12,7 @@ protocol = git
 repo_url = https://github.com/ufs-community/ufs-weather-model
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = b5a1976
+hash = a1143cc
```

Comment:
This hash is not compiled for AQM with the current module files or cmake config.